pyanaconda/modules/storage/partitioning/interactive_utils.py

```diff
@@ -78,3 +78,24 @@ def collect_unused_devices(storage):
     ]
 
     return unused + incomplete + unsupported
+
+
+def collect_bootloader_devices(storage, drive):
+    """Collect the bootloader devices.
+
+    :param storage: an instance of Blivet
+    :param drive: a name of the bootloader drive
+    :return: a list of devices
+    """
+    devices = []
+    boot_drive = drive
```

Comment:
Shouldn't we use drive or name the argument boot_drive?

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2040/commits/f2864948936ef064709ddcc21dc1e0bf388c4441.